esi_leap/objects/contract.py

```diff
@@ -99,3 +118,22 @@ def expire(self, context=None):
         # expire contract
         self.status = statuses.EXPIRED
         self.save(context)
+
+    # @classmethod
+    # def split(cls, context, c):
+    #     # split contract with multiple offers into serval contracts
+    #     offers = c['offers']
+    #     contracts = []
+    #     for o_uuid in offers:
+    #         contract_dict = {}
+    #         o = esi_leap.objects.offer.Offer.get(context, o_uuid).to_dict()
+    #         contract_dict['offer_uuid'] = o['uuid']
+    #         contract_dict['project_id'] = o['project_id']
```

Comment:
This will have to change - and may require an additional query to get the project_id from the bid.